guides/common/modules/con_pulp-data-considerations.adoc

```diff
@@ -0,0 +1,16 @@
+:_mod-docs-content-type: CONCEPT
+
+[id="pulp-data-considerations"]
+= Pulp data considerations
+
+You can clone {Project} server without including Pulp data.
+However, for your cloned environment to work, you do require Pulp data.
+If the target server does not have Pulp data, it is not a fully working {Project}.
+
+To transfer Pulp data to a target server, you have two options:
+
+* Clone using backup with Pulp data
+* Clone using backup without Pulp data and copy `/var/lib/pulp` manually from the source server.
+
```

Comment:
```suggestion
+
```

Maybe add the following explanation directly beneath the second option. Your call.